lib/core/src/sdk.rs

```diff
@@ -556,6 +557,7 @@ impl LiquidSdk {
             let mut current_bitcoin_block: u32 = 0;
             let mut shutdown_receiver = cloned.shutdown_receiver.clone();
             let mut interval = tokio::time::interval(Duration::from_secs(10));
+            #[cfg(not(all(target_family = "wasm", target_os = "unknown")))]
             interval.set_missed_tick_behavior(MissedTickBehavior::Skip);
```

Comment:
If we directly reference the type we can get rid of the conditional on the top

```suggestion
            interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
```

Reply:
Fixed [ca33d13](https://github.com/breez/breez-sdk-liquid/pull/806/commits/ca33d131e64ecba5db6338aa16aa8a062dd9ec28)